lib/fastly/acl_entry.rb

```diff
@@ -0,0 +1,28 @@
+class Fastly
+  class ACLEntry < Base
+    attr_accessor :service_id, :acl_id, :entry_id
+
+    ##
+    # :attr: service_id
+    #
+    # The id of the service this belongs to.
+
+    ##
+    # :attr: acl_id
+    #
+    # The ACL id associated to a set of ACL entries
+
+    ##
+    # :attr: entry_id
+    #
+    # The ACL entry id for a specific entry
+
+    def self.singular
```

Comment:
I believe we want to use `singularize` here as the naming is inspired by `ActiveSupport::Inflector`:
http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-singularize